test/test.js

```diff
@@ -182,6 +183,15 @@ function expectCreatedShot(driver, creator) {
   });
 }
 
+function verifyShotUrl(shotUrl) {
+  const restUrl = shotUrl.substr(backend.length);
+  const hostname = new URL(backend).hostname;
```

Comment:
`new URL()` throws in a lot of cases. It might make sense to try/catch and rethrow with an error like "backend is not a valid URL, please check the value of `process.env.SCREENSHOTS_BACKEND`"

Reply:
We won't arrive here if the backend URL is not valid.  I will put in a validation earlier.